src/Sectiontittle.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./Sectiontittle.css";
+
+function Sectiontittle(props){
```

Comment:
please use object de structuring. Replace props with `{  title }` and replace `{ props.title }` with `{ title }` in line 6.